grid/social-media/posts-comments/profile.supr

```diff
@@ -0,0 +1,130 @@
+name = "social-media/posts-comments"
+version = "1.0.0"
+
+"""
+Get comments on a social media post
+"""
+usecase GetPostComments safe {
+  input {
+    """
+    ID
+    Identifier of the post.
+    """
+    id! string!
+    """
+    Page
+    Pagination cursor (obtained from `previousPage` or `nextPage`).
+    """
+    page string
+  }
+
+  result {
+    """
+    Previous page
+    Previous page cursor.
+    """
+    previousPage string
+    """
+    Next page
+    Next page cursor.
+    """
+    nextPage string
+    """
+    Comments
+    List of comments under the post.
+    """
+    comments [Comment]
+  }
+  
+  error {
+    """
+    Title
+    A short, human-readable summary of the problem type.
+    """
+    title!
+
+    """
+    A human-readable explanation specific to this occurrence of the problem.
+    """
+    detail
+  }
+
+  example success {
+      input {
+          id = "be7cdc05-4264-4d47-bc1f-af704d6abf32"
+      }
+
+      result {
+          comments = [
+            {
+                id = "123"
+                created = {
+                    by = "@alberteinstein"
+                    at = 1652271236
+                }
+                text = "Hello, World!"
+            },
+            {
+                id = "124"
+                created = {
+                    by = "@isaacnewton"
+                    at = 1652272236
+                }
+                lastModified = {
+                    by = "@moderator"
+                    at = 1652272256
+                }
+                text = "(deleted.)"
+            }
+        ]
+      }
+  }
+}
+
+model Comment {
+    """
+    ID
+    Identifier of the comment.
+    """
+    id! string!
+
+    """
+    Created
+    Information about the creation of a comment (author, time).
+    """
+    created {
+        """
+        Created by
+        Identification of the author of the comment.
+        """
+        by string
+        """
+        Created at
+        UNIX timestamp (in seconds) of the comment creation.
+        """
+        at number
+    }
+
+    """
+    Last Modified
+    Information about the last known modification of a comment (author, time).
+    """
+    lastModified {
+        """
+        Created by
+        Identification of the user who last modified the comment.
+        """
+        by string
```

Comment:
Not sure if this information is available everywhere (doubt it)

Reply:
Which is why it's optional and nullable (?)

Reply:
(Also I just noticed the docstring mismatch at L114 🤦 )